OSSM-3252: Service Mesh 2.4 Release Notes (#58437)
openshift/openshift-docs: modules/ossm-rn-fixed-issues.adoc

[id="ossm-rn-fixed-issues-ossm_{context}"]
== {SMProductShortName} fixed issues

> * https://issues.redhat.com/browse/OSSM-3993[OSSM-3993] Previously, Kiali only supported OpenShift OAuth via a proxy on the standard HTTPS port of `443`. Now, Kiali supports it over a non-standard HTTPS port. To enable it, you must define `spec.server.web_port` in the Kiali CR to the proxy's non-standard HTTPS port.

Reviewer: As per this link https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts, i would suggest to use before this update and with this update format, instead of using the current format.

Please apply this change globally for this particular section.

```suggestion
* https://issues.redhat.com/browse/OSSM-3993[OSSM-3993] Before this update, Kiali only supported OpenShift OAuth via a proxy on the standard HTTPS port of `443`. With this update, Kiali supports OpenShift OAuth over a non-standard HTTPS port. To enable the port, you must set the `spec.server.web_port` field to the proxy's non-standard HTTPS port in the Kiali CR.
```

* https://issues.redhat.com/browse/OSSM-2335[OSSM-2335] Dragging the mouse pointer over the Traces scatterchart plot sometimes caused the Kiali console to stop responding due to concurrent backend requests.

> * https://issues.redhat.com/browse/OSSM-2221[OSSM-2221] Gateway injection does not work in control plane namespace. If you use the Gateway injection feature to create a gateway in the same location as the control plane, the injection now succeeds on v2.4 SMCP. The gateway is created in the same location as the control plane.

Author: Done

* https://issues.redhat.com/browse/OSSM-2335[OSSM-2335] Dragging the mouse pointer over the Traces scatterchart plot sometimes caused the Kiali console to stop responding due to concurrent backend requests.

* https://issues.redhat.com/browse/OSSM-2221[OSSM-2221] Previously, gateway injection in the `ServiceMeshControlPlane` namespace was not possible because the `ignore-namespace` label was applied to the namespace by default. 
+
When creating a v2.4 control plane, the namespace no longer has the `ignore-namespace` label applied, and gateway injection is possible.
+
In the following example, the `oc label` command removes the `ignore-namespace` label from a namespace in an existing deployment:
+

> `oc label namespace <istio_system> maistra.io/ignore-namespace-`

Reviewer 2: This command needs to be formatted as a standard, terminal code block: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output
